src/main/java/duke/common/Const.java

```diff
@@ -0,0 +1,13 @@
+package duke.common;
+
+import java.time.format.DateTimeFormatter;
+
+public final class Const {
```

Comment:
I like the use of another class to declare constants and to format date/time. I have not seen this before! 🤔 👍